script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -525,18 +532,20 @@ I had three minutes in total before my body and consciousness to shut down due t
 [sha:9a6561de7b3b8e6f76e04f645fe9983f8576ae7a]{
 -- Page 591, Offset 10208.
 -- 満足に活動できるリミットは１分もない。
-I don't even have one minute left.
+// 05_00_ARC05_9B has him saying fairly explicitly that he has 2 min left,
+// so is this actually 'one full minute has already become nai'?
```

Comment:
I left a comment about that in 5_9B but nobody responded and I forgot to follow-up. I'm gonna _guess_ he's saying "I've already lost one minute" but that timeline doesn't line up either? I guess either way it makes _more_ sense this way

Reply:
Yeah the time seems fucky, but this is at least 3 minutes -> down one minute -> 2 minutes left, even if there's a decent bit of file between (3-1) minutes and (2) minutes

Reply:
makes sense